docs/content/en/docs/tasks/packages/cluster-autoscaler/_index.md

```diff
@@ -0,0 +1,108 @@
+---
+title: "Cluster Autoscaler"
+linkTitle: "Add Cluster Autoscaler"
+weight: 13
+date: 2022-10-20
+description: >
+  Install/upgrade/uninstall Cluster Autoscaler
+---
+
+{{< content "../prereq.md" >}}
+
+
+## Install
+
+<!-- this content needs to be indented so the numbers are automatically incremented -->
+1. Generate the package configuration
+   ```bash
+   eksctl anywhere generate package cluster-autoscaler --source cluster > cluster-autoscaler.yaml
```

Comment:
I think this is outdated, but we will do a bulk update, so feel free to leave it. Commenting just as FYI.

Reply:
```suggestion
   eksctl anywhere generate package cluster-autoscaler --cluster clusterName > cluster-autoscaler.yaml
```